lib/linguist/languages.yml

```diff
@@ -3475,6 +3475,17 @@ UnrealScript:
   tm_scope: source.java
   ace_mode: java
 
+Ur/Web:
```

Comment:
This needs to be the same as the directory for the samples. So we can't use `/`.
Could it be UrWeb or maybe just Ur?